app/views/smart_proxies/index.html.erb

```diff
@@ -1,24 +1,27 @@
+<% javascript('statuses') %>
 <% title _("Smart Proxies") %>
 
 <% title_actions display_link_if_authorized(_("New Smart Proxy"), hash_for_new_smart_proxy_path),
                  documentation_button('4.3SmartProxies') %>
 
-<table class="table table-bordered table-striped table-two-pane table-fixed">
+<table class="table table-bordered table-striped table-fixed">
   <thead>
     <tr>
+      <th class="col-md-1"><%= _('Status') %></th>
       <th class="col-md-2"><%= s_("SmartProxy|Name") %></th>
-      <th class="col-md-5"><%= s_("SmartProxy|Url") %></th>
-      <th class="col-md-3"><%= _("Features") %></th>
-      <th></th>
+      <th class="col-md-4"><%= s_("SmartProxy|Url") %></th>
+      <th class="col-md-5"><%= _("Features") %></th>
     </tr>
   </thead>
   <tbody>
     <% for proxy in @smart_proxies %>
       <tr>
-        <td class="display-two-pane ellipsis"><%= link_to_if_authorized proxy.name, hash_for_edit_smart_proxy_path(:id => proxy).merge(:auth_object => proxy, :authorizer => authorizer) %></td>
+        <td class="ellipsis">
+          <span class="proxy-status" data-url="<%= ping_smart_proxy_path(proxy) %>"><%= image_tag '/assets/spinner.gif' %> <%= _("Connecting..") %></span>
```

Comment:
Ok I think the spinner is enough, spinner *and* Connecting text is too much. It also reformats the width of the table when all proxies are done (status icons are just 16px or something like that. In other words, keep only the spinner.

During my initial testing, I don't see the spinner for some reason. Maybe asset cache on development setup?